src/client/updater.py

```diff
@@ -22,23 +22,28 @@ class UpdateSettings:
 
     def should_notify(self, releases, force=True):
         self._logger.debug(releases)
-        have_server = 'server' in releases
-        have_stable = 'stable' in releases
-        have_pre = 'pre' in releases
-        have_beta = 'beta' in releases
-
         if force:
-            return have_server or have_stable or have_pre or have_beta
-        else:
-            current_version = Version(config.VERSION)
-            # null out build because we don't care about it
-            current_version.build = ()
+            return True
+
+        server_releases = [release for release in releases if release['branch']=='server']
+        stable_releases = [release for release in releases if release['branch']=='stable']
+        pre_releases = [release for release in releases if release['branch']=='pre']
+        beta_releases = [release for release in releases if release['branch']=='beta']
 
-            notify_stable = have_stable and self.updater_branch == UpdateBranch.Stable.name and Version(releases['stable']['new_version']) > current_version
-            notify_pre = have_pre and self.updater_branch == UpdateBranch.Prerelease.name and Version(releases['pre']['new_version']) > current_version
-            notify_beta = have_beta and self.updater_branch == UpdateBranch.Unstable.name and Version(releases['beta']['new_version']) > current_version
+        have_server = len(server_releases) > 0
```

Comment:
We can as well rely on the fact that empty lists evaluate to false and do 'if server_releases'.